pages/_app.tsx

```diff
@@ -1,8 +1,9 @@
-import '../styles/output.css'
-import type { AppProps } from 'next/app'
+import "../styles/globals.css";
+import "../styles/output.css";
+import type { AppProps } from "next/app";
 
 function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+	return <Component {...pageProps} />;
```

Comment:
stredniky by nemely existovat :)))

Reply:
je to tu nejako nekonzistentne, ale zrejme tu viac nie su ako su. takze mozno by som si predsa len mohol dovolit pridat jedno pravidlo do prettier :)

Reply:
jj, myslim ze dneska je v TS skoro nikdo nepise :)

Reply:
zalozil jsem dotaznik na slacku,
mne osobne se lepe cte se strednikama, vim kde to konci...